EvotoApi/App_Start/Startup.Auth.cs

```diff
@@ -27,6 +27,10 @@ public void ConfigureAuth(IAppBuilder app)
             {
                 AuthenticationType = DefaultAuthenticationTypes.ApplicationCookie,
                 LoginPath = new PathString("/manage/login"),
+                SlidingExpiration = true,
+                CookieHttpOnly = true,
```

Comment:
I apparently totally fucked my understanding off what this does. I'm right as in it'll cover HTTP/HTTPS, but it actually stops the client from interacting with cookies. :+1: TIL